syntax = "proto3";

message DocumentParserRequest {
    bytes file_content = 1;
}

message DocumentParserResponse {
    map<int32, string> pages = 1;
    map<string, string> metadata = 2;
}

service DocumentParserService {
    rpc processDocument (DocumentParserRequest) returns (DocumentParserResponse);
}
